packages/flutter_webrtc/tizen/inc/flutter_webrtc_base.h

```diff
@@ -0,0 +1,204 @@
+#ifndef FLUTTER_WEBRTC_BASE_HXX
+#define FLUTTER_WEBRTC_BASE_HXX
+
+#include <flutter/encodable_value.h>
+#include <flutter/event_channel.h>
+#include <flutter/event_stream_handler_functions.h>
+#include <flutter/method_channel.h>
+#include <flutter/plugin_registrar.h>
+#include <flutter/standard_message_codec.h>
+#include <flutter/standard_method_codec.h>
+#include <flutter/texture_registrar.h>
+#include <string.h>
+
+#include <list>
```

Comment:
nit
```cpp
#include <flutter/texture_registrar.h>

#include <string.h>
#include <list>
```

Reply:
string.h → string

Reply:
I have removed not used head file :
```cpp
#include <flutter/texture_registrar.h>
#include <string.h>
```

Reply:
This file makes use of std::string. (https://google.github.io/styleguide/cppguide.html#Include_What_You_Use)

Reply:
The code has been copied from https://github.com/flutter-webrtc/flutter-webrtc/tree/main/common/cpp/include. We may just ignore style rule violations and keep the files as they are.

Reply:
I mean, the current code is just an old copy of the upstream code and will be updated soon. Please see the above comment.

Reply:
Ok please ignore comment about `flutter_common.h`. But it looks like the refactored code still needs `<string.h>` header.
https://github.com/flutter-webrtc/flutter-webrtc/commit/8f183ed17a3f0d75eb05c397bb88cc60312d7127#diff-e6e2895c9ae54220a68383cabc3b678a8b50299ab8f269e6962678c09857c57cR6

My first review was that empty space should be above <string.h>. I thought it was a simple typo.
And <string.h> was deleted.
If the code is to be modified by refactoring anyway, I don't mind leaving it as it is.